flocx_market/common/exception.py

```diff
@@ -0,0 +1,34 @@
+class MarketplaceException(Exception):
+    msg_fmt = "An unknown exception occurred."
+    code = 500
+    safe = False
+
+    def __init__(self, message=None, **kwargs):
+        self.kwargs = kwargs
```

Comment:
We don't appear to be doing anything with `self.kwargs` after this (we're using `kwargs` on line 16).